WebRoot/css/zm.css

```diff
@@ -4021,23 +4021,69 @@ HTML>BODY .appt-selected .appt_allday_body
 }
 
 /* ZmChooseFolderDialog and ZmPickTagDialog */
+
+#ChooseFolderDialog {
+	@ChooseFolderDialog@
+}
+
+#ChooseFolderDialog .WindowOuterContainer {
+	@ChooseFolderDialogContainer@
+}
+
 .ChooserDialog {
-	width:320px;
+	@ChooseFolderDialogBody@
 }
 
 .ChooserDialog .DwtInputField {
-	padding-bottom:3px;
-	padding-top:3px;
+	@ChooseFolderDialogInputContainer@
 }
 
-.ChooserDialog INPUT {
-	width:100%;
+#ChooseFolderDialog .ChooserDialog INPUT {
+	@ChooseFolderDialogInputField@
 }
 
 .ChooserDialog .overview {
-	background-color:white;
-	border:1px solid black;
-	overflow:auto;
+	@ChooseFolderDialogFolderContainer@
+}
+
+.ChooserDialog .dialogOverview {
+	@ChooseFolderDialogFolderContainerContent@
+}
+
+.ChooserDialog .DwtTreeItem {
+	@ChooseFolderDialogFolderItem@
+}
+
+.ChooserDialog .DwtTreeItemLevel1ChildDiv {
+	padding-left:0px;
```

Comment:
The main idea of having property is reusability, which in this case in not going to be.
skin.css would ideally contains changes related to skins, which is not the case here.
So, kept this in zm.css